packages/front-end/services/datasources.ts

```diff
@@ -720,43 +722,125 @@ interface InitialDatasourceResources {
   }[];
 }
 
-export function getInitialDatasourceResources({
-  datasource,
+const getClickHouseInitialDatasourceResources = ({
+  table,
+  organization,
 }: {
-  datasource: DataSourceInterfaceWithParams;
-}): InitialDatasourceResources {
-  if (
-    datasource.type === "bigquery" &&
-    datasource.settings?.schemaFormat === "ga4"
-  ) {
-    const params = datasource.params;
+  table: string;
+  organization: string;
+}): InitialDatasourceResources => {
+  return {
+    factTables: [
+      {
+        factTable: {
+          name: "Clickhouse Events",
+          description: "",
+          sql: `SELECT
+  timestamp,
+  user_id,
+  user_attributes_json,
+  event_name,
+  geo_country,
+  geo_city,
+  geo_lat,
+  geo_lon,
+  ua_device_type,
+  ua_browser,
+  ua_os,
+  utm_source,
+  utm_medium,
+  utm_campaign,
+  url_path,
+  session_id
+FROM ${table} WHERE organization = '${organization}'`,
```

Comment:
Ideally table would always just be `events` here.  I'd like to avoid using the full qualified table path if possible since it's very opaque to the user (they don't know their org/datasource id).  As a user, I would want to just be able to run `select * from events` and have it work.